doc/README.tpl.md

```diff
@@ -279,17 +290,17 @@ Now if the json contains extra properties, the test will fail.
 
 Available matchers are:
 
-| matcher    | description                       |
-|----------- |---------------------------------- |
-| `match`    | property must match given regexp  |
-| `matches`  | see `match`                       |
-| `contain`  | property must contain given value |
-| `contains` | see `contain`                     |
-| `defined`  | property must not be `undefined`  |
-| `present`  | see `defined`                     |     
-| `equal`    | property must equal given value   |
-| `equals`   | see `equal`                       |     
-
+| matcher                   | description                       |
+|-------------------------- |---------------------------------- |
+| `match`                   | property must match given regexp  |
+| `matches`                 | see `match`                       |
+| `contain`                 | property must contain given value |
+| `contains`                | see `contain`                     |
+| `defined`                 | property must not be `undefined`  |
+| `present`                 | see `defined`                     |     
+| `equal`                   | property must equal given value   |
+| `equals`                  | see `equal`                       |     
+| `equalRelativeDate`       | see `equalRelativeDate`           |
```

Comment:
I think you're meant to write `see XXX` when it's a wording variation. Here you have to describe it.

Reply:
yep done